Command/ExpiredVideoNotificationCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Pumukit\ExpiredVideoBundle\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+
+class ExpiredVideoNotificationCommand extends ContainerAwareCommand
+{
+    private $dm = null;
+    private $mmobjRepo = null;
+    private $type = "expired";
+    private $user_code = 'owner';
```

Comment:
@Yurujai, @rubenrua 
This value should be read from `'pumukitschema.personal_scope_role_code'` parameter:
https://github.com/campusdomar/PuMuKIT2/blob/2.3.x/src/Pumukit/SchemaBundle/DependencyInjection/Configuration.php#L57-L60
https://github.com/campusdomar/PuMuKIT2/blob/2.3.x/src/Pumukit/SchemaBundle/DependencyInjection/PumukitSchemaExtension.php#L33